designer/client/components/ComponentCreate/ComponentCreateList.tsx

```diff
@@ -59,13 +57,17 @@ export const ComponentCreateList = ({ onSelectComponent }: Props) => {
                 >
                   {i18n(`fieldTypeToName.${component.type}`)}
                 </a>
+                <p className="govuk-body-s">
```

Comment:
I think we should be using `<div className="govuk-hint">{i18n(...)}</div>` for the hints, so they follow GOV UK style guide